src/Libraries/Microsoft.Extensions.Http.Resilience/Hedging/HttpHedgingStrategyOptions.cs

```diff
@@ -16,8 +17,9 @@ public class HttpHedgingStrategyOptions : HedgingStrategyOptions<HttpResponseMes
     /// Initializes a new instance of the <see cref="HttpHedgingStrategyOptions"/> class.
     /// </summary>
     /// <remarks>
-    /// By default, the options are set to handle only transient failures,
-    /// that is, timeouts, 5xx responses, and <see cref="HttpRequestException"/> exceptions.
+    /// By default, the options are configured to handle only transient failures.
+    /// Specifically, this includes HTTP status codes 408, 429, 500 and above, 
+    /// as well as <see cref="HttpRequestException"/> and <see cref="TimeoutRejectedException"/> exceptions.
```

Comment:
`BrokenCircuitException` is also considered a [transient error](https://github.com/dotnet/extensions/blob/05df0833c0b889bd8f795e14643efa270dfdbd1e/src/Libraries/Microsoft.Extensions.Http.Resilience/Hedging/HttpClientHedgingResiliencePredicates.cs#L53C13-L53C44).